features/providers/civil_application_journey.feature

```diff
@@ -267,6 +267,25 @@ Feature: Civil application journeys
     Then I click 'Save and continue'
     Then I should be on a page showing 'Check your answers'
 
+  @javascript @vcr
+  Scenario: I want to change email address from the about financial assessment page
+    Given I complete the journey as far as check your answers
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'must complete a financial assessment'
+    Then I click 'Continue'
+    Then I should be on a page showing 'Does your client use online banking?'
+    Then I choose "Yes"
+    Then I click 'Save and continue'
+    Then I am on the About the Financial Assessment page
+    And I click link 'Change'
+    Then I should be on a page showing 'Email address'
+    Then I fill 'email' with 'test@test.com'
+    Then I click 'Save and continue'
+    Then I am on the About the Financial Assessment page
+    Then I should be on a page showing 'test@test.com'
+
+
+
```

Comment:
For consistency, I would just leave 1 space between this scenario and the next instead of 3 because.
That's what we do everywhere